packages/fxa-content-server/app/styles/modules/_custom-rows.scss

```diff
@@ -248,9 +248,13 @@ input[type='checkbox'] {
 }
 
 .two-col-items {
-  column-width: 160px;
+  display: grid;
```

Comment:
Are we all good with this on the browser support front then? What times we live in!

Reply:
Exciting times indeed: https://caniuse.com/#search=grid Grid started to be supported w/ Fx 52, which just happens to be our current ESR - 1.